packages/rum-core/src/common/constants.js

```diff
@@ -56,6 +61,11 @@ const RESOURCE_INITIATOR_TYPES = [
   'iframe'
 ]
 
+/**
+ * The amount of time it is allowed for a transaction to be reused in another startTransaction
+ */
+const REUSABILITY_THRESHOLD = 10000
```

Comment:
Any reasoning behind this particular value?

Reply:
I'll make the value into ms.